libcxx/include/__ranges/transform_view.h

```diff
@@ -173,7 +173,7 @@ template <input_range _View, copy_constructible _Fn>
 #  endif
   requires __transform_view_constraints<_View, _Fn>
 template <bool _Const>
-class transform_view<_View, _Fn>::__iterator : public __transform_view_iterator_category_base<_View, _Fn> {
+class transform_view<_View, _Fn>::__iterator : public __transform_view_iterator_category_base<_View, _Fn, _Const> {
```

Comment:
It seems that we can use `__maybe_const<_Const, _Fn>` here and avoid modifying `__transform_view_iterator_category_base`.